packages/graphql_client/lib/models/twins.dart

```diff
@@ -0,0 +1,625 @@
+    this.relay = false,
+    this.twinID = false,
+  });
+
+  @override
+  String toString() {
+    String returnOptions = "";
+    returnOptions = _addToReturnList(returnOptions, "id", id);
+    returnOptions = _addToReturnList(returnOptions, "accountID", accountID);
+    returnOptions = _addToReturnList(returnOptions, "gridVersion", gridVersion);
+    returnOptions = _addToReturnList(returnOptions, "publicKey", publicKey);
+    returnOptions = _addToReturnList(returnOptions, "relay", relay);
+    returnOptions = _addToReturnList(returnOptions, "twinID", twinID);
+    if (returnOptions == "") returnOptions = "id \n";
+    return returnOptions;
+  }
+}
+
+class TwinConnectionsEdgeReturnOptions {
+  bool cursor;
+  TwinReturnOptions? node;
+
+  TwinConnectionsEdgeReturnOptions({
+    this.cursor = false,
+    this.node,
+  });
+
+  @override
+  String toString() {
+    String returnOptions = "";
+    if (!cursor && node == null) node = TwinReturnOptions(id: true);
+
+    returnOptions = _addToReturnList(returnOptions, "cursor", cursor);
+    returnOptions = _addToReturnList(returnOptions, "node {", node != null);
+    returnOptions =
+        _addToReturnList(returnOptions, node.toString(), node != null);
+    returnOptions = _addToReturnList(returnOptions, "} ", node != null);
+    if (returnOptions != "") returnOptions = "edges { \n $returnOptions } \n";
+    return returnOptions;
+  }
+}
+
+class TwinConnectionsPageReturnOptions {
+  bool endCursor;
+  bool hasNextPage;
+  bool hasPreviousPage;
+  bool startCursor;
+
+  TwinConnectionsPageReturnOptions({
+    this.endCursor = false,
+    this.hasNextPage = false,
+    this.hasPreviousPage = false,
+    this.startCursor = false,
+  });
+
+  @override
+  String toString() {
+    String returnOptions = "";
+    returnOptions = _addToReturnList(returnOptions, "endCursor", endCursor);
+    returnOptions = _addToReturnList(returnOptions, "hasNextPage", hasNextPage);
+    returnOptions =
+        _addToReturnList(returnOptions, "hasPreviousPage", hasPreviousPage);
+    returnOptions = _addToReturnList(returnOptions, "startCursor", startCursor);
+    if (returnOptions != "") {
+      returnOptions = "pageInfo { \n $returnOptions } \n";
+    }
+    return returnOptions;
+  }
+}
+
+class TwinConnectionsReturnOptions {
+  TwinConnectionsEdgeReturnOptions? edges;
+  TwinConnectionsPageReturnOptions? pageInfo;
+  bool totalCount;
+
+  TwinConnectionsReturnOptions({
+    this.edges,
+    this.pageInfo,
+    this.totalCount = false,
+  });
+
+  @override
+  String toString() {
+    String returnOptions = "";
+    if (edges != null) returnOptions += edges.toString();
+    if (pageInfo != null) returnOptions += pageInfo.toString();
+    returnOptions = _addToReturnList(returnOptions, "totalCount", totalCount);
+    if (returnOptions == "") {
+      returnOptions += '''
+      edges {
+      node {
+        id
+      }
+    }
+''';
+    }
+    return returnOptions;
+  }
+}
+
+class TwinInfo {
```

Comment:
I think it should be `NodeInfo`, as when u add these in graphql they're under property named `node`.